seed_message_sender/settings.py

```diff
@@ -124,7 +136,6 @@
 
 # REST Framework conf defaults
 REST_FRAMEWORK = {
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAdminUser',),
```

Comment:
This is removing a duplicated directive that is defined a few lines below and uses the correct class: rest_framework.permissions.IsAuthenticated